lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_translations_action.rb

```diff
@@ -0,0 +1,86 @@
+# This action is the new version of android_update_metadata (AndroidUpdateMetadataAction) and should now be used instead of that one
+
+module Fastlane
+  module Actions
+    class AndroidDownloadTranslationsAction < Action
+      def self.run(params)
+        require_relative '../../helper/android/android_localize_helper.rb'
+        require_relative '../../helper/git_helper.rb'
+
+        res_dir = File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'src', 'main', 'res')
```

Comment:
I think it'd be great if this could be provided as a param to avoid the need to rely on a bunch of environment variables being present

Reply:
Yeah I didn't want to include that refactor just yet because so far all our other actions are relying on `PROJECT_ROOT_FOLDER` and `PROJECT_NAME` so didn't want to start diverging here.

But I like the idea of already starting to migrate away from `ENV['PROJECT_NAME']`, so will probably add a `ConfigItem` whose corresponding `env_name` is `PROJECT_NAME`, picking the best of both worlds

Reply:
Fixed in b0e155af23eb84e6d7fc10e55965ea834e37267d